tutorial/markdown/java/spring-boot/spring-boot.md

```diff
@@ -2,12 +2,12 @@
 # frontmatter
 path: "/tutorial-quickstart-java-springboot"
 # title and description do not need to be added to markdown, start with H2 (##)
-title: Start with Java and Spring Boot
+title: Quickstart in Couchbase with Java and Spring Boot
 short_title: Java and Spring Boot
 description:
-  - Learn how to configure Spring Data with Couchbase
+  - Learn how to configure Spring Boot with Couchbase
   - Explore key-based operations and SQL++ querying using Spring Data Couchbase repositories
```

Comment:
This is the same as CRUD.

Reply:
now adapted from python and golang tutorials